### ISSUE TITLE
vmalert will not start with multiple rules files in the same dir

### ISSUE THREAD

**Issue by dakotacody** [VictoriaMetrics/VictoriaMetrics#708]
**Describe the bug**

Trying to add multiple rule files to a directory vmalert will not start and throws an misleading error. 
For instance, this command: 
`./vmalert-prod -rule /root/alertmanager/rules/victoria-metrics/*yml -rule /root/alertmanager/rules/monitoring/*yml -rule /root/alertmanager/rules/bfy-cass/*yml -datasource.url=http://localhost:9481  -notifier.url=http://10.40.155.120:9093,http://10.40.155.11:9093     -remoteWrite.url=http://localhost:10480     -remoteRead.url=http://localhost:9481    -evaluationInterval=60`

Throws this error: 
`2020-08-19T19:42:38.852Z	fatal	VictoriaMetrics/app/vmalert/main.go:64	failed to init: failed to init datasource: datasource.url is empty`

After troubleshooting and inspecting the yaml in the files most recently added: 

I moved one of the files out of the way and it started right up 
`stg-1:~/alertmanager/rules/monitoring# mv ../hbase-rules.yml . `

I then put the file back and the error reoccurred. 

I then moved the _other_ file out of the way and it started up fine.  
`stg-1:~/alertmanager/rules/monitoring# mv ../system-checks.yml . `

Since it starts successfully with either of the two rule files in place, the problem does not seem to lie in the contents but rather the existence of more than one rule file being in the directory. 

**To Reproduce**
Steps to reproduce the behavior
Place two rule files in a directory and specify your rule to use that dir with a wildcard like so: 
` -rule /root/alertmanager/rules/monitoring/*yml`

**Expected behavior**
Rules should load as indicated from the vmalert -help 
  -rule array
    	Path to the file with alert rules. 
    	Supports patterns. Flag can be specified multiple times. 
    	Examples:
    	 -rule /path/to/file. Path to a single file with alerting rules
    	 -rule dir/*.yaml -rule /*.yaml. Relative path to all .yaml files in "dir" folder, 
    	absolute path to all .yaml files in root.
    	Rule files may contain %{ENV_VAR} placeholders, which are substituted by the corresponding env vars.
    	Supports array of values separated by comma or specified via multiple flags.

**Version**
```./vmalert-prod -version 
vmalert-20200815-130850-tags-v1.40.0-0-ged00eb3f3
```
**Used command-line flags**
-rule /root/alertmanager/rules/bfy-cass/*yml 
-datasource.url=http://localhost:9481  
-notifier.url=http://10.40.155.120:9093,http://10.40.155.11:9093     
-remoteWrite.url=http://localhost:10480     
-remoteRead.url=http://localhost:9481
-evaluationInterval=60

**Comment by hagen1778** [comment]
Hi @dakotacody!
It is likely that flag values weren't correctly parsed because of asterisk character - try to enclose values in quotes:
```
./bin/vmalert-prod -rule="/root/alertmanager/rules/victoria-metrics/*yml" -rule "/root/alertmanager/rules/monitoring/*yml" -rule "/root/alertmanager/rules/bfy-cass/*yml" -datasource.url=http://localhost:9481 -notifier.url=http://10.40.155.120:9093,http://10.40.155.11:9093 -remoteWrite.url=http://localhost:10480 -remoteRead.url=http://localhost:9481 -evaluationInterval=60s
```

**Comment by dakotacody** [comment]
This works now... I actually was going to try that but the example from the -help didn't show them so I hadn't.   I see you've got a different issue you are already addressing that in.  

Thanks for the speedy reply!

**Comment by valyala** [comment]
FYI, this should be fixed in [v1.40.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
